src/DataModel/IResource.ts

```diff
@@ -0,0 +1,5 @@
+export interface IResource
+{
+    "@id": string;
+    "@type"?: Array<string>
```

Comment:
No - currently I'm using JSON-LD framing to transform hypermedia controls I discover to an internal model. It is rooted in the hydra's structure, but still it is supposed to be separated from the JSON-LD  serialization.